text/0018-export_behavior.md

```diff
@@ -0,0 +1,145 @@
+- Feature Name: Export Behavior
+- Start Date: <!-- (fill me in with today's date, YYYY-MM-DD) -->
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Export functionality exists in qri, but is lacking many options and pieces of functionality.
+
+# Motivation
+[motivation]: #motivation
+
+An <a href="https://github.com/qri-io/rfcs/blob/master/text/0014-export.md">earlier RFC</a> began describing the export functionality. However, it left a number of details unspecified, which led to too much uncertainty to move forward with implementation. While it did mention many of the options for export, it left out discussions of defaults, the interactions between options, and basically how `export` is expected to work. This RFC hopes to remedy this situation so that implementation can proceed without hesitation.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Export is a piece of functionality that will generate the version of a dataset which is external to qri storage, so that it can live instead on a local filesystem. The goal is to have a document defintion of the structured data that has been, up until now, represented only internally by qri. Here are some of the use cases that `export` is meant to handle:
+
+* Getting a local binary copy of a dataset, such that it can be safetly deleted from qri, and later imported without any loss of fidelity or history.
+* Exporting an entire dataset with metadata and body to analyze in a custom script. A single file "json" or "yaml", for example.
+* Converting to a binary format (like xlsx) for use in another program like Excel or OpenOffice.
+* Deciding to stop using qri, and wanting to get all of your data out in a high-fidelity form, with both machine and human readable formats.
+
+These should all be doable using `export`.
+
+Some non-goals:
+
+* Retrieving only a single component of a dataset, like the title.
+* Converting a dataset's body to a different format than it is being stored as (csv to json or cbor).
+
+These types of operations should be added to `get` instead, which already has similar functionality.
```

Comment:
I feel like we should should note that what binds all these use cases together is an effort to have a complete  dataset. `get` is intended for working with components of datasets. I feel like it should be a non-goal of export to support flags for removing parts of datasets from export, and should instead focus on having that functionality in `get`